app/components/ZoomPanel.tsx

```diff
@@ -0,0 +1,180 @@
+import { VStack } from "@chakra-ui/react";
+import {
+  Tooltip,
+  ButtonGroup,
+  Icon,
+  IconButton,
+  CompassIcon,
+  AddIcon,
+  MinusIcon,
+} from "@nycplanning/streetscape";
+import { LinearInterpolator } from "@deck.gl/core";
+import type { MapViewState } from "@deck.gl/core";
+
+type zoomProps = {
+  viewState: MapViewState;
+  setViewState: (next: MapViewState) => void;
+  step?: number;
+  minZoom?: number;
+  maxZoom?: number;
+};
+
+export function ZoomPanel({
+  viewState,
+  setViewState,
+  step = 1,
+  minZoom = 10,
+  maxZoom = 20,
+}: zoomProps) {
+  const clamp = (value: number) => Math.min(maxZoom, Math.max(minZoom, value));
+
+  const animate = (partial: Partial<MapViewState>) =>
+    setViewState({
+      ...viewState,
+      ...partial,
+      transitionDuration: 500,
+      transitionInterpolator: new LinearInterpolator(["zoom", "bearing"]),
+    });
+
+  const zoomBy = (delta: number) =>
+    animate({ zoom: clamp(viewState.zoom + delta) });
+  const resetCompass = () => animate({ bearing: 0 });
+
+  return (
+    <VStack rowGap={"0"}>
+      <ButtonGroup isAttached={true} orientation="vertical" spacing={0}>
+        <Tooltip
```

Comment:
These tooltips don't seem to ever close once opened on mobile with touch events. Since you can't have hover effects on touch devices, I think she should just never show up on them?

Reply:
There seems to be inconsistent behavior depending on the browser:
 - FF mobile emulator, the tooltips remain
 - Chrome and Safari, they do close after clicks/touches.

Should we get rid of the tooltips altogether @TylerMatteo?

Reply:
I will remove the tooltips. It doesn't seem to work on mobile as I'd hoped it would.